### DISCUSSION TITLE
Dependabot/dependency graph - transitive dependencies

### DISCUSSION THREAD

**Discussion by javixeneize** [community/community#21630] [New to GitHub]
<p>Hi</p>
<p>I have a question about transitive dependencies. Are those displayed in dependabot or dependency graph? I can only see the first level dependencies analysed but nothing else in a test project i have created – <a href="https://github.com/javixeneize/dependatest/network/dependencies" class="inline-onebox" rel="noopener nofollow ugc">Dependencies · javixeneize/dependatest · GitHub</a></p>
<p>Thanks</p>

**Comment by ernest-phillips** [comment]
<aside class="quote no-group" data-username="javixeneize" data-post="1" data-topic="178432">
<div class="title">
<div class="quote-controls"></div>
 javixeneize:</div>
<blockquote>
<p>transitive dependencies</p>
</blockquote>
</aside>
<p>The dependency graph does not handle these types of dependencies.<br>
Dependencies will need to be explicitly stated in order for them to be captured by the graph.</p>

**Comment by javixeneize** [comment]
<p>Thats what i feared. Then, this is not of much use for java applications, isnt it?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p><a href="https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem">We support dependency <em><strong>updates</strong></em> for repositories using Gradle</a> as well, but we’re still working on overall dependency graph support.</p>
<p>We do have an open internal issue requesting such functionality though; I’ve included your feedback there. I can’t make any guarantees on a release or timeframe, but keep an eye on <a href="https://github.blog/changelog/">our changelog</a> for updates!</p>

**Comment by javixeneize** [comment]
<p>Ah right, if this is in the backlog and you are planning to fix it I’m happy with that :wink:</p>
<p>Thanks!</p>

**Comment by ljharb** [comment]
Unfortunately https://github.com/github/roadmap/issues/494#issuecomment-2529161577 doesn't have any details, really - what makes a feature "under review by the product team", and what exempts a feature from that category?

One of the biggest security problems in open source is that the vast majority of transitive CVEs are de facto false positives due to the code paths they're used with and/or the context (dev vs prod, etc) they're used in, and surfacing this graph information would be a really useful way to cut down on the noise.
